app/lib/coursesHelper.coffee

```diff
@@ -1,6 +1,21 @@
 Levels = require 'collections/Levels'
 utils = require 'core/utils'
 
+courseShortName = (course) ->
+  switch
+    when /game-dev/.test(course.get('slug')) then 'GD'
+    when /web-dev/.test(course.get('slug')) then 'WD'
+    else 'CS'
+
+shortenCourseName = (course) ->
```

Comment:
The names usually have a number if it is needed except "Introduction to Computer Science".
Therefore if no number is found, assume it is the first, or 1.

Reply:
These two method names are confusing. Neither of them return a short name; really, they return an acronym, and one includes the number and the other doesn't. One is a noun and the other a verb...